fitbenchmarking/cli/main.py

```diff
@@ -239,7 +240,18 @@ def run(problem_sets, results_directory, options_file='', debug=False):
             groups=groups,
             group_link=group_links,
             zip=zip))
-    webbrowser.open_new(output_file)
+
+    # Finds the file prefix required for Mac
+    prefix = "file:///" if output_file[0] != "/" else "file://"
```

Comment:
On WSL I get the following for this:
```
INFO:
The FitBenchmarking results will be placed into the folder:

   /home/alister/fitbenchmarking/data_assim

To change this use the -r or --results-dir optional command line argument. You can also set 'results_dir' in an options file.
Start : This command cannot be run due to the error: The system cannot find the file specified.
At line:1 char:1
+ Start "file:///home/alister/fitbenchmarking/data_assim/results_index. ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [Start-Process], InvalidOperationException
    + FullyQualifiedErrorId : InvalidOperationException,Microsoft.PowerShell.Commands.StartProcessCommand


INFO:
The FitBenchmarking results have been opened in your browser from this url:

   file:///home/alister/fitbenchmarking/data_assim/results_index.html
```

Removing the last `file://` seems to make it work.
In this case, if I set DISPLAY to "", I get the following:
```
INFO:
The FitBenchmarking results will be placed into the folder:

   /home/alister/fitbenchmarking/data_assim

To change this use the -r or --results-dir optional command line argument. You can also set 'results_dir' in an options file.
Start : This command cannot be run due to the error: The system cannot find the file specified.
At line:1 char:1
+ Start "/home/alister/fitbenchmarking/data_assim/results_index.html"
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [Start-Process], InvalidOperationException
    + FullyQualifiedErrorId : InvalidOperationException,Microsoft.PowerShell.Commands.StartProcessCommand


INFO:
The FitBenchmarking results have been opened in your browser from this url:

   /home/alister/fitbenchmarking/data_assim/results_index.html
```

I think this is because with `DISPLAY=:0` it trys to use the windows hosts file system... in that case having the prefix of "W:/" might work although it is impossible to know this from our end...